src/Internal/Calculator/BcMathCalculator.php

```diff
@@ -42,15 +43,25 @@ public function mul(string $a, string $b) : string
      */
     public function divQ(string $a, string $b) : string
     {
-        return \bcdiv($a, $b, 0);
+        $result = \bcdiv($a, $b, 0);
+        if ($result === null) {
+            throw DivisionByZeroException::divisionByZero();
+        }
```

Comment:
These assertions are not necessary. The Calculator interface [explicitly states](https://github.com/brick/math/blob/15ebb638d309aff44a016f3fcf8c2d83b7bef611/src/Internal/Calculator.php) that all inputs must be validated, or the result is undefined. The applies to `Calculator::divQ()`, which states:

> The divisor, must not be zero.

This is for performance, and is indirectly largely covered by the large number of tests!

Reply:
Thanks, didn't notice that. Removed the check, put the error is the baseline file for now (we can cast the function result to string to get rid of the error, but I don't want to do it just for static analyzer).